Framework/Core/src/DataProcessingDevice.cxx

```diff
@@ -2093,6 +2095,11 @@ bool DataProcessingDevice::tryDispatchComputation(ServiceRegistryRef ref, std::v
           ref.get<CallbackService>().call<CallbackService::Id::PostProcessing>(o2::framework::ServiceRegistryRef{ref}, (int)action.op);
           dpContext.postProcessingCallbacks(processContext);
           streamContext.postProcessingCallbacks(processContext);
+          auto& info = o2::framework::ServiceRegistryRef{ref}.get<TimingInfo>();
+          if (info.timeslice <= 1652945069870351) {
```

Comment:
What is this number?

Reply:
Ok. We should probably have some information in TimingInfo to keep track of what triggered the timeslice.

Reply:
indeed, would make sense. Since I copied and pasted that number from the callbackpolicy.cxx :)